penv.pl

```diff
@@ -85,4 +87,4 @@
 if [ -n "$BASH" -o -n "$ZSH_VERSION" ] ; then
     hash -r
 fi
-eval $(perl -I{{ PENV }}/perl5/lib/perl5 -Mlocal::lib={{ PENV }})
+eval $(perl -I{{ PENV }}/lib/perl5 -Mlocal::lib={{ PENV }})
```

Comment:
This is maybe not a typo, but due to bootstrapping `local::lib` from home directory?